scipy/interpolate/ndgriddata.py

```diff
@@ -50,11 +53,11 @@ class NearestNDInterpolator(NDInterpolatorBase):
 
     """
 
-    def __init__(self, x, y, rescale=False):
+    def __init__(self, x, y, rescale=False, cKDTree_options=dict()):
```

Comment:
Please avoid using mutable default arguments. Better use `options=None`, and then `if options is None: options = dict()`.